# path_to_robot_pose_launch: "/home/vlad/Desktop/camera_ws/src/aruco_robot/launch/robot_pose.launch"

robot_pose_publisher:
  robot_marker_id: 1

camera_pose_covariance: [
  1000, 0, 0, 0, 0, 0,
  0, 1000, 0, 0, 0, 0,
  0, 0, 1000, 0, 0, 0,
  0, 0, 0, 1000, 0, 0,
  0, 0, 0, 0, 1000, 0,
  0, 0, 0, 0, 0, 1000
]